### ISSUE TITLE
Fallback fonts not sized properly

### ISSUE THREAD

**Issue by UtkarshVerma** [bakkeby/dwm-flexipatch#296]
I have an alias in my `~/.config/fontconfig/fonts.xml` like this:

```xml
    <alias>
        <family>monospace</family>
        <prefer>
            <family>Comic Code Ligatures</family>
            <family>Symbols Nerd Font</family>
        </prefer>
    </alias>
```

This is simply setting Nerd Font symbols as a fallback in case the glyphs aren't found in my main monospace font, i.e. Comic Code Ligatures.

This works properly in `st`, however, dwm fails to size the fallback font properly and the glyphs are rendered very small.

Here are a few screenshots.

`static const char *fonts[]               = { "monospace-12", "Symbols Nerd Font-14" };`
![image](https://user-images.githubusercontent.com/31820255/188078807-9f45329f-20c1-4d5c-8109-a67f6c430dcf.png)

`static const char *fonts[]               = { "monospace-12" };`
![image](https://user-images.githubusercontent.com/31820255/188079012-d619780d-3e17-42b9-baec-b05c74ae07f9.png)

These is the output of st using the same font family without FONT2 patch:
![image](https://user-images.githubusercontent.com/31820255/188080741-a5ff5180-952c-4e05-a2d1-d1b1a4489835.png)

**Comment by bakkeby** [comment]
I don't think that this would be the responsibility of dwm to figure out. If it needs to look up additional fonts then it will just use the same size as for the primary font, and not all fonts are created alike, like some fonts have space around the characters while other does not. Specifying the font size using pixel size rather than size may also make a difference. 

There is also #255 where the monospaced version of the nerd font has small icons compared to the non-monospaced version of the font.

**Comment by UtkarshVerma** [comment]
Yeah, using pixelsize did make a difference, but the vertical alignment got messed up.

![image](https://user-images.githubusercontent.com/31820255/188278745-1d5a771a-3f5c-43ba-bf2b-0bc594fa7848.png)

This seems really weird.
